bin/p2-inspect/main.go

```diff
@@ -51,7 +51,7 @@ func main() {
 	opts := kp.Options{
 		Address: *consulUrl,
 		Token:   *consulToken,
-		Client:  net.NewHeaderClient(*headers),
+		Client:  net.NewHeaderClient(*headers, nil),
```

Comment:
I'd feel more comfortable explicitly passing `http.DefaultTransport`, but I won't press that point.